### DISCUSSION TITLE
Should I not disclose the chromatic project token?

### DISCUSSION THREAD

**Discussion by k35o** [storybookjs/storybook#26535] [Help] [answered]
### Summary

When I set up chromatic, the project token is written in package.json, and when I run `@chromatic-com/storybook`, the project token is written in `chromatic.config.json`, is it ok to publish these? Is it OK to disclose these to the public?

If this is not possible, how should I hide the chromatic.config.json?

**Comment by valentinpalkovic** [comment, marked as answer]
For open source projects, you can use an env variable like described here https://www.chromatic.com/docs/cli/#configuration to set it locally as well as on CI.

**Comment by k35o** [comment]
@valentinpalkovic 
Thanks!I miss it.
